src/components/calcite-pagination/calcite-pagination.tsx

```diff
@@ -66,11 +66,17 @@ export class CalcitePagination {
   //
   //--------------------------------------------------------------------------
 
-  /** Emitted whenever the selected page changes.
-   * @event calcitePaginationUpdate
+  /**
+   * Emitted whenever the selected page changes.
+   * @deprecated use calcitePaginationChange instead
```

Comment:
oh nice, does this give a warning when used by end devs?

Reply:
That'd be rad, but no. It just makes its way to the doc.

![Screen Shot 2021-02-05 at 11 14 05 AM](https://user-images.githubusercontent.com/197440/107078657-77130f00-67a3-11eb-80fe-658a2e7af745.png)